code/datums/diseases/advance/symptoms/flesh_eating.dm

```diff
@@ -56,15 +57,32 @@ Bonus
 			Flesheat(M, A)
 
 /datum/symptom/flesh_eating/proc/Flesheat(mob/living/M, datum/disease/advance/A)
-	var/get_damage = rand(15,25) * power
-	M.take_overall_damage(brute = get_damage, required_status = BODYPART_ORGANIC)
-	if(pain)
-		M.adjustStaminaLoss(get_damage * 2)
-	if(bleed)
-		if(ishuman(M))
-			var/mob/living/carbon/human/H = M
-			H.bleed_rate += 5 * power
-	return 1
+	if(ishuman(M))
+		var/mob/living/carbon/human/H = M
+		H.bleed_rate += 2 * power //bleeding is quite strong. this is more than enough 
+		H.bleed(max(10*power, H.bleed_rate))//having power actually up the bleed rate on this puts it into a pretty dangerous territory. this should be more managable
+		H.add_splatter_floor(H.loc)
+		if(bleed) // this is really, really messy
+			var/geysers = rand(2, 6)
+			var/bloodsplatters = transmission
+			var/list/splatterdirs = list()
+			var/turf/T = H.loc
+			playsound(T, 'sound/effects/splat.ogg', 50, 1)
+			H.visible_message("<span class='danger'>Blood bursts from [H]'s flesh!</span>", \
+	  "<span class='userdanger'>Blood spews forth from your flesh! It hurts!</span>")
+			for(var/i=0 to geysers)
```

Comment:
```suggestion
			for(var/i in 1 to geysers)
```

Reply:
This was how i originally coded it. it doesn't work. I changed it and tested it just now to be sure- only the i=0 works, don't ask me why

Reply:
update: issue was elsewhere, fixed, works like this, i dumb